protocol-spec/enumeration.md

```diff
@@ -176,6 +239,43 @@ system. Can happen at any time after identification, as it is assumed
 many server implementations will support devices with hotplugging  
 capabilities that do not require specific scanning/discovery sessions.
 
+**Introduced In Version:** 0
+
+**Message Version:** 1
+
+**Fields:**
+
+* _Id_ \(unsigned int\): Message Id
+* _DeviceName_ \(string\): Descriptive name of the device
+* _DeviceIndex_ \(unsigned integer\): Index used to identify the device
+  when sending Device Messages.
+* _DeviceMessages_ \(dictionary\): Accepted Device Messages 
+    * Keys \(string\): Type names of Device Messages that the device will accept
+    * Values \(dictionary\): Attributes for the Device Messages. Both keys and values are strings.
```

Comment:
Same comment as above. I really think we can make these dicts with variable value types.